000_START_HERE.md

```diff
@@ -57,15 +57,15 @@ You will need to engage with three parts of it:
 * The shell panel to the right, where you'll run code.
 
 To set up and learn how to use the interface, please follow
-the [replit IDE setup video](https://youtu.be/c0GdcmPfKL4).
+the [replit IDE setup video](https://vimeo.com/954334352/1f0dee9379).
 This video has subtitles.
 
 You may find the default replit theme hard to read. We have
 created two themes which combine good contrast and
 readability. I would recommend you use one of them:
 
-* [Makers Reader Theme (Light)](https://replit.com/theme/@training17/makers-reader)
-* [Makers Reader Theme (Dark)](https://replit.com/theme/@training17/makers-reader-dark)
+* [Makers Reader Theme (Light)](https://replit.com/theme/@Makers-Academy/makers-reader)
```

Comment:
[question] Should these be published?

Reply:
Thank you! Didn't know there was the concept of theme publication :)

All done now